packages/create-plugin/templates/github/workflows/release.yml

```diff
@@ -13,14 +13,17 @@ permissions: read-all
 jobs:
   release:
     permissions:
+      id-token: write
       contents: write
+      attestations: write
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: grafana/plugin-actions/build-plugin@main
-        # Uncomment to enable plugin signing
-        # (For more info on how to generate the access policy token see https://grafana.com/developers/plugin-tools/publish-a-plugin/sign-a-plugin#generate-an-access-policy-token)
-        #with:
+        with:
+          attestation: true
```

Comment:
I'm sure this was already asked within the team but will this cause issues with private repositories?

Reply:
I was thinking about the same thing. What would happen if you run this in a private repo?

Reply:
`To use artifact attestations in private or internal repositories, you must be on a GitHub Enterprise Cloud plan.`

that's what github says.